catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,13 +28,17 @@
                     availability="optional"/>
 
     <reference id="encryptionService" interface="ddf.security.encryption.EncryptionService"/>
+    <reference id="clientFactoryFactory" interface="org.codice.ddf.cxf.client.ClientFactoryFactory"/>
```

Comment:
❓ Technically the corresponding feature should depend on the security bundle providing this factory right? Technically that should be true for the other service references in this blueprint, so that's a little out of scope of this PR.

Reply:
True, but it will be available by the time someone attempts to create an instance of the source.

Reply:
Yes it is. I'm just making a plug for another instance where we're not following best practices and making independent, encapsulated bundles 😄